pepper-apis/src/main/resources/db-changes/seed/DDP-5514-add-translations-de-hi-it-ja-pl-tr.xml

```diff
@@ -0,0 +1,667 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="ypavlov" id="20210111-update-translations-de">
+        
```

Comment:
Since these are platform-level translations, it might affect other studies that uses these languages. @charlottedye do you see anything here that might affect Prion?

Reply:
@yufengwng These 6 languages were introduced for ATCP, so I'm pretty sure it won't affect the others.
BTW Translations provided by Language Scientific.

Reply:
https://broadinstitute.atlassian.net/browse/DDP-5332

Reply:
Missed this earlier.  Yep, none of these languages are used for Prion.